GOVERNANCE.md

```diff
@@ -318,17 +407,17 @@ If the pull request submitter is not a Core Maintainer, the PR can be closed by
-3. A non-trivial, significant change should be discussed within the RFC GitHub issue and approved during an RFC meeting call.
-4. During an RFC meeting, the moderator will attempt to achieve consensus on the RFC proposal.
+2. A trivial change can be discussed and approved entirely within the RFC GitHub issue, as long as there are no objections from Core or TSC members. This is not considered a formal vote.
+3. A non-trivial, significant change should be discussed within the RFC and approved during an RFC meeting call. In some cases, an RFC may be approved outside of an RFC meeting using Pull Request reviews as a proxy for votes.
+4. During an RFC meeting, the person leading the call will attempt to achieve consensus on the RFC proposal.
 5. **If consensus is reached:** the RFC is approved.
-6. **If consensus is not reached:** Maintainers must make all reasonable attempts to resolve issues and reach consensus in GitHub or a follow-up RFC meeting. The process of reaching consensus can take time, and should not be rushed as long as all participants are making a reasonable effort to respond.
-7. **If consensus still cannot be reached:** The project Steward may initiate the first fallback mechanism by limiting the vote to Core Maintainers.
-8. **If consensus still cannot be reached:** The project Steward may initiate a final fallback vote of Core Maintainers, of which an overwhelming majority (80%+) is required to pass.
-9. **If consensus still cannot be reached:** The RFC is closed without approval.
+6. **If consensus is not reached:** The RFC author and TSC members must make all reasonable attempts to resolve issues and reach consensus in GitHub or a follow-up RFC meeting. The process of reaching consensus can take time, and should not be rushed as long as all participants are making a reasonable effort to respond.
+7. **If consensus still cannot be reached:** The project Steward may invoke [rough consensus](https://en.wikipedia.org/wiki/Rough_consensus) to resolve an RFC that has not achieved absolute consensus, as described below (borrowed from the [IETF](https://datatracker.ietf.org/doc/html/rfc2418)):
+
+> Working groups make decisions through a "rough consensus" process. Astro consensus does not require that all participants agree although this is, of course, preferred. In general, the dominant view of the TSC shall prevail. (However, "dominance" is not to be determined on the basis of volume or persistence, but rather a more general sense of agreement). Consensus can be determined by a show of hands, humming, or any other means on which the TSC agrees (by rough consensus, of course). Note that 51% of the TSC does not qualify as "rough consensus" and 99% is better than rough. It is up to the project Steward to determine if rough consensus has been reached.
```

Comment:
Who determines rough consensus if an RFC is discussed on an RFC meeting w/o the Steward present?

I suppose that just comes down to what sort of RFC it is, how fundamental it is?

Reply:
It's up the the steward to determine if rough consensus is reached, so they would have to be present in any case of non-absolute consensus.

In practice, all of our votes so far have been through absolute consensus (100%), and any objections have been addressable through follow-up and working together between the RFC champion and the objector. So this is a bit of an edge case, but still a good one to plan for.

Reply:
This extends to the "let's wait a moment for someone to object to consensus" done in RFC calls traditionally, right?

Reply:
yup!